nibabel/brikhead.py

```diff
@@ -0,0 +1,499 @@
+
+        return ''.join(orient)
+
+    def get_space(self):
+        """Returns space of dataset
+
+        Returns
+        -------
+        space : str
+        """
+        listed_space = self.info.get('TEMPLATE_SPACE', 0)
+        space = space_codes.label[listed_space]
+
+        return space
+
+    def get_affine(self):
+        """ Returns affine of dataset
+
+        Examples
+        --------
+        >>> header = AFNIHeader(parse_AFNI_header('example4d+orig.HEAD'))
+        >>> header.get_affine()
+        array([[ -3.    ,  -0.    ,  -0.    ,  49.5   ],
+               [ -0.    ,  -3.    ,  -0.    ,  82.312 ],
+               [  0.    ,   0.    ,   3.    , -52.3511],
+               [  0.    ,   0.    ,   0.    ,   1.    ]])
+        """
+        # AFNI default is RAI-/DICOM order (i.e., RAI are - axis)
+        # need to flip RA sign to align with nibabel RAS+ system
+        affine = np.asarray(self.info['IJK_TO_DICOM_REAL']).reshape(3, 4)
+        affine = np.row_stack((affine * [[-1], [-1], [1]],
+                               [0, 0, 0, 1]))
+
+        return affine
+
+    def get_data_scaling(self):
+        """ AFNI applies volume-specific data scaling
+
+        Examples
+        --------
+        >>> header = AFNIHeader(parse_AFNI_header('example4d+orig.HEAD'))
+        >>> header.get_data_scaling()
+        """
+        floatfacs = self.info.get('BRICK_FLOAT_FACS', None)
+        if floatfacs is None or not np.any(floatfacs):
+            return None
+        scale = np.ones(self.info['DATASET_RANK'][1])
+        floatfacs = np.asarray(floatfacs)
+        scale[floatfacs.nonzero()] = floatfacs[floatfacs.nonzero()]
+
+        return scale
+
+    def get_slope_inter(self):
+        """Use `self.get_data_scaling()` instead"""
+        return None, None
+
+    def get_data_offset(self):
+        """Data offset in BRIK file"""
+        return DATA_OFFSET
+
+    def get_volume_labels(self):
+        """Returns volume labels
+
+        Returns
+        -------
+        labels : list of str
+
+        Examples
+        --------
+        >>> header = AFNIHeader(parse_AFNI_header('example4d+orig.HEAD'))
+        >>> header.get_volume_labels()
+        ['#0', '#1', '#2']
+        """
+        labels = self.info.get('BRICK_LABS', None)
+        if labels is not None:
+            labels = labels.split('~')
+
+        return labels
+
+
+class AFNIImage(SpatialImage):
+    """AFNI Image file
+
+    Can be loaded from either the BRIK or HEAD file (but MUST specify one!)
+
+    Examples
+    --------
+    >>> brik = load('example4d+orig.BRIK')
+    >>> brik.shape
+    (33, 41, 25, 3)
+    >>> brik.affine
+    array([[ -3.    ,  -0.    ,  -0.    ,  49.5   ],
+           [ -0.    ,  -3.    ,  -0.    ,  82.312 ],
+           [  0.    ,   0.    ,   3.    , -52.3511],
+           [  0.    ,   0.    ,   0.    ,   1.    ]])
+    >>> head = load('example4d+orig.HEAD')
+    >>> (head.get_data() == brik.get_data()).all()
```

Comment:
Breaking on memmap for old numpy. The following should work, though you may need to add an `import numpy as np` at the top of the doctest.

```Python
>>> np.array_equal(head.get_data(), brik.get_data())
True
```